custom/lib/Passage.php

```diff
@@ -299,9 +300,21 @@ public function updateUser(
         trigger_error('Method ' . __METHOD__ . ' is deprecated', E_USER_DEPRECATED);
 
         try {
-            return $this->user->update($user_id, $update_user_request);
+            $args = $this->cast($update_user_request, UpdateUserArgs::class);
+            return $this->user->update($user_id, $args);
         } catch (PassageError $e) {
             throw $e->getPrevious();
         }
     }
+
+    private function cast(object $object, string $class): mixed
```

Comment:
I'm confused by this. Why is it was expecting one class but receiving another? For instance, why would createUser not expect `CreateUserRequest`? This is the class that is genreated.

Reply:
it's because of the class aliasing. the new functions take a different class name. in the other languages it was as easy as type aliasing but i tried php's `class_alias`, which didn't seem to work to allow an end user to use the aliased class names. so i had to create new classes that are just copies of the generated classes with different names. because of that, it was necessary to cast the classes otherwise it threw an error at runtime